app.js

```diff
@@ -88,27 +89,54 @@ app.post('/hangouts', (request, response) => {
 	}).catch(error(response));
 });
 
-app.post('/reportEvent', (request, response) => {
+app.post('/reportEvent', async (request, response) => {
+	const fhirUrl = request.body.fhirUrl;
+	if (!fhirUrl) {
+		response.status(400).send("missing fhirUrl");
+		return;
+	}
+	if (!settings.authorizedFhirUrls.includes(fhirUrl)) {
+		response.status(401).send(`unauthorized fhirUrl ${fhirUrl}`);
```

Comment:
401 actually represents "unauthenticated" - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401 which isn't appropriate here because it's an authorization issue not authentication issue.

403 or 400 would be better.

Reply:
Done.